src/main/java/org/folio/template/util/TemplateEngineHelper.java

```diff
@@ -51,10 +51,10 @@ public static Response mapExceptionToResponse(Throwable throwable) {
         .build();
     }
 
-    Future<Response> validationFuture = Future.future();
-    ValidationHelper.handleError(throwable, validationFuture.completer());
-    if (validationFuture.isComplete()) {
-      Response response = validationFuture.result();
+    Promise<Response> promise = Promise.promise();
+    ValidationHelper.handleError(throwable, promise);
+    if (promise.future().isComplete()) {
+      Response response = promise.future().result();
```

Comment:
Can you add a unit test for this in TemplateEngineHelperTest? SonarCloud fails this pull request because "70.3% Coverage on New Code (is less than 80%)".

Reply:
added